packages/salesforcedx-vscode-core/src/commands/templates/executors/LibraryForceApexClassCreateExecutor.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { DirFileNameSelection } from '@salesforce/salesforcedx-utils-vscode';
+import { ApexClassOptions, TemplateType } from '@salesforce/templates';
+import { nls } from '../../../messages';
+import { LibraryBaseTemplateCommand } from '../libraryBaseTemplateCommand';
+import { APEX_CLASS_TYPE } from '../metadataTypeConstants';
+
+export class LibraryForceApexClassCreateExecutor extends LibraryBaseTemplateCommand<
```

Comment:
Have the executor and main command function both exported from the same file leads to sadness in unit test land so I moved the executor for each of these out to their own file.